packages/backend/src/plugins/catalog.ts

```diff
@@ -33,6 +34,10 @@ class OpenshiftResourceProcessor implements CatalogProcessor {
 
     if (originLocation.type === 'rhacm-managed-cluster') {
       resource.metadata.annotations ||= {};
+      resource.metadata.annotations['prometheus.io/alerts'] = 'all';
+      resource.metadata.annotations[
+        'prometheus.io/labels'
+      ] = `managed_cluster_id=${resource.metadata.annotations[ANNOTATION_CLUSTER_ID]}`;
```

Comment:
```suggestion
	  const clusterId = resource.metadata.annotations[ANNOTATION_CLUSTER_ID]
      resource.metadata.annotations['prometheus.io/labels'] = `managed_cluster_id=${clusterId}`;
```

Reply:
Should be resolved :+1: + I have bumped dependencies and I have added `@janus-idp/backstage-plugin-ocm-common` as dependency since it was not there and was neccessary for `ANNOTATION_CLUSTER_ID`